.github/workflows/fluent.syntax.yml

```
@@ -17,17 +18,16 @@ jobs:
         java-version: 1.8
     - name: Build and check
       working-directory: ./fluent.syntax
-      run: ./gradlew check
+      run: ./gradlew test
   lint:
+    name: lint Kotlin
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
-    - name: install
-      run: brew install ktlint
-    - name: ktlint
-      working-directory: ./fluent.syntax
-      run: ktlint
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build and check
+        working-directory: ./fluent.syntax
+        run: ./gradlew ktlintCheck
```

Align the indention here

```suggestion
    - uses: actions/checkout@v2
    - name: Set up JDK 1.8
      uses: actions/setup-java@v1
      with:
        java-version: 1.8
    - name: Build and check
      working-directory: ./fluent.syntax
      run: ./gradlew ktlintCheck
```